client/src/screens/search-groups.screen.js

```diff
@@ -67,23 +65,12 @@ class Group extends Component {
   constructor(props) {
     super(props);
     this.myGroups = this.props.myGroups;
-    this.joinGroup = this.joinGroup.bind(this);
-    this.groupUpdate = this.props.groupUpdate.bind(this);
+    this.navigation = this.props.navigation;
```

Comment:
bind goToGroup:
```
this.goToGroup = this.goToGroup.bind(this);
```

Reply:
how come? goToGroup is a function not a prop

Reply:
dw, answered my own question